File: cocos2dx-js/Poco/sdk/Attributor.js
var Attributor = function () {

}

Attributor.prototype.getAttr = function (node, attrName) {
    var node_ = node
    if (!node.__isPocoNodeWrapper__) {
        node_ = node[0]
    }
    return node_.getAttr(attrName)
}
    
Attributor.prototype.setAttr = function (node, attrName, attrVal) {
    var node_ = node
    if (!node.__isPocoNodeWrapper__) {
        node_ = node[0]
    }
    node_.setAttr(attrName, attrVal)
}

try {
    module.exports = Attributor;
} catch (e) {
    if (window.module && window.module.exports) {
        window.module.exports = Attributor;
    }
}

File: sdk/lua/sdk/Attributor.lua
local Attributor = {}
Attributor.__index = Attributor

function Attributor:getAttr(node, attrName)
    if node.__isPocoNodeWrapper__ == nil then
        node = node[1]
    end
    return node:getAttr(attrName)
end

function Attributor:setAttr(node, attrName, attrVal)
    if node.__isPocoNodeWrapper__ == nil then
        node = node[1]
    end
    node:setAttr(attrName, attrVal)
end

return Attributor